man/systemctl.xml

```diff
@@ -2117,6 +2117,15 @@ Jan 12 10:46:45 example.com bluetoothd[8900]: gatt-time-server: Input/output err
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--reboot-argument=</option></term>
+
+        <listitem>
+          <para>When used with <command>reboot</command>,
+          the argument is passed as the positional argument.</para>
```

Comment:
nah, the positional argument has been removed.... Actually I think the blurb about "firmware specific" from the "reboot" command explanation should be moved here, and the "reboot" command doc should just briefly mention that the switch exists

Reply:
I've just pushed a new commit. Can you please let me know, if that is what you meant here?